info/v2/container.go

```diff
@@ -85,7 +85,7 @@ type ContainerSpec struct {
 	Image string `json:"image,omitempty"`
 }
 
-type ContainerStats struct {
+type DeprecatedContainerStats struct {
```

Comment:
While I like this change, I wonder if it would be better to split it into a separate PR? Both because it seems a little unrelated to the disk stats, and we could make sure the disk changes are not blocked while dealing with merge conflicts etc. from the API change. If you think it's easier to keep it in this PR, I won't stop you...